newsrevenuehub/rev-engine
# refs #NRH-309 Refactor of thumbnail generation.

#### What's this PR do?
This PR is a bugfix to hopefully address the problems we were seeing with the thumbnail name rejected by the system.
To that end, refactors the generation of thumbnails to remove some of the `sorl` obscurity. Splits images and thumbnails into their own locations in the media store.

#### How should this be manually tested?
Add images to the sidebar of a DonationPage. They should now save and display both thumbnail and image.

#### Do any changes need to be made before deployment to staging? production? (adding environment variables, for example)?
No.
